net/pdnsd/files/etc/config/pdnsd.conf

```diff
@@ -0,0 +1,24 @@
+global {
+	perm_cache=1024;
+	cache_dir="/tmp/pdnsd";
+	run_as="nobody";
+	server_ip = 127.0.0.1;
+	server_port = 7453;
+	status_ctl = on;
+	query_method = tcp_only;
```

Comment:
Is this for querying upstream servers?  If that is the case, I understand what you intend to achieve, but still this is not sane default to start with.

Reply:
have /etc/pdnsd.conf.example

Reply:
?  `tcp_only` may suit your need, but it's not general works-out-of-the-box option.